etl/steps/data/meadow/technology/2023-03-06/mobile_money.meta.yml

```diff
@@ -0,0 +1,33 @@
+dataset:
+  namespace: technology
+  short_name: mobile_money
+  title: Global Mobile Money Dataset (GSMA, 2022)
+  description: |-
+    The Global Mobile Money Dataset is a set of global metrics for the mobile money industry based on data collected and analyzed by the GSMA Mobile Money program.
+
+    The GSMA Mobile Money program considers services that meet the following definitions:
+
+    • The service must be available to the unbanked, e.g., people who do not have access to a formal account at a financial institution.
+    • The service must offer at least one of the following services: Storage of value; Domestic or international transfer; mobile payment, including bill payment, bulk disbursement, and merchant payment.
+    • The service must offer a network of physical transactional points outside bank branches and ATMs, making the service widely accessible to everyone.
+    • The service must offer an interface for initiating transactions for agents or customers that is available on mobile devices.
+    • Mobile banking services that offer the mobile phone as just another channel to access a traditional banking product are not included.
+    • Payment services linked to a traditional banking product or credit cards, such as Apple Pay and Google Wallet, are not included.
+  sources:
+    - name: GSMA (2022)
```

Comment:
Maybe it would be good to have this as what GSMA stands for although I see from their website they try hard not to make this obvious, so perhaps the acronym is more recognisable in this case?